src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -280,6 +319,15 @@ class CoreWorkerDirectActorTaskSubmitter
   /// exceeds this quantity. This threshold is doubled each time it is hit.
   int64_t next_queueing_warn_threshold_;
 
+  /// The event loop where the actor task events are handled.
+  instrumented_io_context actor_task_io_service_;
```

Comment:
is this only used for testing? we should consider doing it differently; add testing code/public test function is A LOT of overhead to read.

Reply:
Removed.